trestle/core/commands/author/profile.py

```diff
@@ -86,17 +86,18 @@ def _init_arguments(self) -> None:
             default=False
         )
         self.add_argument('-s', '--sections', help=const.HELP_SECTIONS, required=False, type=str)
+        self.add_argument('-rs', '--required-sections', help=const.HELP_REQUIRED_SECTIONS, required=False, type=str)
 
     def _run(self, args: argparse.Namespace) -> int:
         try:
             log.set_log_level_from_args(args)
-            trestle_root = args.trestle_root
+            trestle_root = pathlib.Path(args.trestle_root)
             if not file_utils.is_directory_name_allowed(args.output):
```

Comment:
I think trestle_root is already of type Path in all commands.

Reply:
Correct.  I have changed it but I give it typing to indicate it is type pathlib.Path